packages/algorand-js/src/internal/interpreter/opcode-list.ts

```diff
@@ -0,0 +1,159 @@
+import { IStack } from "../../lib/stack";
+import { TealError } from "../core/errors";
+import { ERRORS } from "../core/errors-list";
+import type { AppArgs, StackElem } from "../types";
+import { parseNum } from "../utils/bigint";
+import { Op } from "./opcode";
+
+const BIGINT0 = BigInt("0");
+// pops string([]byte) from stack and pushes it's length to stack
+export class Len extends Op {
+  execute (stack: IStack<StackElem>): void {
+    this.assertStackLen(stack, 1);
+    const top = stack.pop();
+    if (typeof top !== 'string') {
```

Comment:
Bytes can't be represent as strings. Bytes must be represented as `Uint8Array`.

Reply:
ok